Person/Person/main.m

```diff
@@ -51,13 +59,55 @@ - (NSString *)phoneNumber {
     return _phoneNumber;
 }
 
-@end
+- (BOOL)checkSameCity:(NSString *)city {
```

Comment:
Ok, yes. Thanks!

I originally had it that way, then I changed it in order to NSLog the
answer Yes or No as a way to check myself. Otherwise it only returned a 1
or 0.

Thanks again!

Justine
On Jun 29, 2015 11:47, "Mike Kavouras" <notifications@github.com> wrote:

> In Person/Person/main.m
> <https://github.com/accesscode-2-2/unit-0-hw-week-2/pull/31#discussion_r33478010>
> :
>
> > @@ -51,13 +59,55 @@ - (NSString *)phoneNumber {
> >      return _phoneNumber;
> >  }
> >
> > -@end
> > +- (BOOL)checkSameCity:(NSString *)city {
>
> This is 100% correct. You could make it a bit more concise by just return
> the result of the isEqualToString method call:
>
> - (BOOL)checkSameCity:(NSString *)city {
>   return [_city isEqualToString:city];
> }
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/accesscode-2-2/unit-0-hw-week-2/pull/31/files#r33478010>
> .
>